src/applications/dto/create-application.dto.ts

```
@@ -0,0 +1,7 @@
+export class CreateApplicationDto {
+     name: string;
```

ESLint

[error] Delete `···`

(prettier/prettier)